templates/entry/meta/slim-meta.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Display meta for slim template.
+ *
+ * @package WPAMPTheme
+ */
+
+$locations = wp_get_post_terms( get_the_ID(), 'location', array(
+	'fields' => 'names',
+) );
+$location  = ! empty( $locations ) ? $locations[0] : '--';
+$reviews   = wp_count_comments( get_the_ID() );
+?>
+<div class="h4 line-height-2">
+	<?php get_template_part( 'templates/entry/meta/slim-rating' ); ?>
+	<?php /* translators: %d: Number of reviews. */ ?>
+	<span class="travel-results-result-subtext mr1"><?php printf( esc_html__( '%d Reviews', 'travel' ), esc_html( $reviews->approved ) ); ?></span>
```

Comment:
How about:
```php
esc_html( _n( '%d Review', '%d Reviews', $reviews->approved, 'travel' ) )
```
in place of:
```php
esc_html__( '%d Reviews', 'travel' )
```

Though it's not part of this PR's diff, could you please make a similar change to [functions.php](https://github.com/xwp/travel/blob/1a4375af8bfb3feeffcf5a5aeee5ae1465901361/includes/functions.php#L126)?

Reply:
Thanks for making these changes!